app/models/container_deployment.rb

```diff
@@ -7,7 +7,7 @@ class ContainerDeployment < ApplicationRecord
   has_many :container_volumes, :as => :parent
   has_many :custom_attributes, :as => :resource, :dependent => :destroy
   has_many :authentications, :as => :resource, :dependent => :destroy
-  DEPLOYMENT_TYPES = %w(origin openshift-enterprise).freeze
+  DEPLOYMENT_TYPES = %w(origin).freeze
```

Comment:
@zeari let's not touch the `container_deployment` part, here it is ok to select which one to deploy.